app/containers/Profile/ProfileForm.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form/immutable';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+
+import {
+  AVATAR_FIELD,
+  DISPLAY_NAME_FIELD,
+  POSITION_FIELD,
+  COMPANY_FIELD,
+  ABOUT_FIELD,
+  LOCATION_FIELD,
+} from './constants';
+
+import renderTextInput from './renderTextInput';
+import renderTextarea from './renderTextarea';
+import renderFileInput from './renderFileInput';
+import renderLocationField from './renderLocationField';
+
+import { imageValidation, strLength20, strLength96 } from './validate';
+
+/* eslint-disable-next-line */
+let ProfileForm = props => {
+  const { handleSubmit, submitting, invalid, sendProps } = props;
+  const { ipfs } = sendProps.profile;
+
+  return (
```

Comment:
As we discussed, let's separate profile form from profile view screen into separate components. Profile should be just displaying data without input fields.